cfme/cloud/tenant.py

```diff
@@ -194,48 +192,97 @@ def is_displayed(self):
             self.entities.breadcrumb.active_location == 'Tag Assignment')
 
 
-class Tenant(Navigatable):
-    _param_name = 'Tenant'
+class TenantCollection(Navigatable):
+    """Collection object for the :py:class:`cfme.cloud.tenant.Tenant`."""
```

Comment:
**REQUIRED**: As per below, We need to have an ```__init__``` which takes appliance and saves it to ```self.appliance```